feature-libs/order/components/order-details/order-detail-billing/order-detail-billing.component.html

```diff
@@ -1,15 +1,27 @@
-<div class="cx-order-items" *ngIf="order$ | async as order">
-  <div class="cx-review-summary" *ngIf="order.paymentInfo as paymentDetails">
-    <div class="cx-review-summary-card cx-review-summary-payment-card">
-      <cx-card
-        [content]="getPaymentMethodCard(paymentDetails) | async"
-      ></cx-card>
-    </div>
+<div *ngIf="true">
```

Comment:
What does it check `*ngIf="true"`? Is it necessary here, maybe I am missing the point.

Reply:
My bad, forgot to remove it from testing.

Reply:
Looks like i forgot to push a commit, the rest of the files are not the latest version neither, reworking it.